daprdocs/layouts/partials/components/state-stores.html

```diff
@@ -0,0 +1,41 @@
+{{- $groups := dict
+" Generic" $.Site.Data.components.state_stores.generic
+"Microsoft Azure" $.Site.Data.components.state_stores.azure
+"Google Cloud Platform (GCP)" $.Site.Data.components.state_stores.gcp
+"Amazon Web Services (AWS)" $.Site.Data.components.state_stores.aws
+"Oracle Cloud" $.Site.Data.components.state_stores.oracle
+}}
+
+{{ range $group, $components := $groups }}
+<h3>{{ $group }}</h3>
+<table width="100%">
+    <tr>
+        <th>Component</th>
+        <th>CRUD</th>
+        <th>Transactional</th>
+        <th>ETag</th>
+        <th>Actors</th>
+        <th>Query</th>
+        <th>Status</th>
+        <th>Component version</th>
+        <th>Since runtime version</th>
+    </tr>
+    {{ range sort $components "component" }}
+    <tr>
+        <td><a href="/reference/components-reference/supported-state-stores/{{ .link }}/" }}>{{ .component
+                }}</a>
+        </td>
+        <td>{{ if .features.crud }}✅{{ end }}</td>
+        <td>{{ if .features.transactions }}✅{{ end }}</td>
+        <td>{{ if .features.etag }}✅{{ end }}</td>
```

Comment:
Looks like you are missing code here for TTL and Actors columns. Am I correct here?

Reply:
The actors column is below, the logical and of etags and transactions (which is what signals actor support, correct?) and yes, looks like I somehow missed out on the TTL column, I will fix that